packages/radix/src/theme.ts

```diff
@@ -105,7 +109,7 @@ export const theme: Theme = {
       'hsl(0, 100%, 10%)',
     ],
   },
-};
+} as const;
```

Comment:
haha brilliant, that's exactly how I was thinking of doing it. Is it working well? I'll do a proper testing later.

Reply:
Absolutely, I didn't run into any issues. Let me know how it works for you 🙌